.travis.yml

```diff
@@ -1,9 +1,5 @@
 language: cpp
 
-sudo: false
-
-cache: apt
```

Comment:
any reason to drop the apt cache?

Reply:
Thanks for asking! I could find no sign in the docs that `cache: apt` is a thing that works. Perhaps it once was but travis removed it? I see they recommend against caching debian packages at https://docs.travis-ci.com/user/caching/#things-not-to-cache